guides/doc-Quickstart/master.adoc

```diff
@@ -21,7 +15,12 @@ It is configurable and the Puppet modules can be read or run in "no-op" mode to
 
 include::common/modules/ref_supported-operating-systems.adoc[]
 
+ifdef::foreman-el,foreman-deb[]
 The installation will require 4GB of memory.
+endif::[]
```

Comment:
```suggestion
The installation requires 4GB of memory.
```

Reply:
Obviously this suggestion should not replace the endif. I couldn't add the suggestion to the actual line.